src/main/java/io/vertx/redis/client/impl/CommandReporter.java

```diff
@@ -22,12 +22,12 @@ enum Tags {
     SPAN_KIND("span.kind", reporter -> "client"),
 
     // DB
-    // See https://github.com/opentracing/specification/blob/master/semantic_conventions.md
+    // See https://opentelemetry.io/docs/specs/semconv/database/redis/
 
     DB_USER("db.user", reporter -> reporter.user),
-    DB_INSTANCE("db.instance", reporter -> reporter.database),
-    DB_STATEMENT("db.statement", reporter -> reporter.command),
-    DB_TYPE("db.type", reporter -> "redis");
+    DB_NAMESPACE("db.namespace", reporter -> reporter.database),
+    DB_QUERY_TEXT("db.query.text", reporter -> reporter.command),
```

Comment:
I think this is actually `db.operation.name`, per the OpenTelemetry docs.

Reply:
Good catch, thanks @Ladicek 

I assumed we had the full command here